tools/enftun-setup

```diff
@@ -137,7 +137,7 @@ del_routes() {
 }
 
 cmd_up() {
-    [[ -z $(ip link show dev "$INTERFACE" 2>/dev/null) ]] || die "$INTERFACE already exists."
+    [[ -z $(f "$INTERFACE" 2>/dev/null) ]] || die "$INTERFACE already exists."
```

Comment:
This change looks like an accidental commit?

Reply:
Oops, sorry, no idea how this happened.